.gitignore

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
can we just copy the content of https://github.com/kubernetes-sigs/kueue/blob/main/.gitignore

Reply:
Done